cmd/cmd.go

```diff
@@ -60,18 +60,34 @@ var rootCmd = &cobra.Command{
 
 var createCmd = &cobra.Command{
 	Use:   "create",
-	Short: "Creates the required tables in PostgreSQL",
+	Short: "Creates the required tables in the database",
 	RunE: func(_ *cobra.Command, _ []string) error {
 		u, err := loadDatabaseURI()
 		if err != nil {
 			return err
 		}
-		pg, err := db.NewPostgreSQL(u, postgresSchema, nil)
-		if err != nil {
+		uri := os.Getenv("DATABASE_URL")
+		if strings.HasPrefix(uri, "mongodb://") {
+			mdb, err := db.NewMongoDB(uri)
+			if err != nil {
+				return err
+			}
+			err = mdb.CreateCollection()
+			if err != nil {
+				return err
+			}
+			defer mdb.Close()
 			return err
+		} else if strings.HasPrefix(uri, "postgres://") || strings.HasPrefix(uri, "postgresql://") {
+			pg, err := db.NewPostgreSQL(u, postgresSchema, nil)
+			if err != nil {
+				return err
+			}
+			defer pg.Close()
+			return pg.CreateTable()
+		} else {
+			return fmt.Errorf("A URL não contém 'mongodb' nem 'postgres'")
```

Comment:
Corrigindo o erro que a CI acusa

```suggestion
			return fmt.Errorf("a URL não contém 'mongodb' nem 'postgres'")
```